src/app/elements/AppElement.js

```diff
@@ -168,6 +172,12 @@ export default class AppElement extends LitElement {
           this.requestUpdate();
         }
         break;
+      case 'save-transform':
+        if (this.panel === 'scene' && this.activeScene === e.detail.uuid){
+          console.log('made it to inside save-transform switch');
```

Comment:
this console log can be removed -- as a side note -- is this ever called? I don't see the dispatchEvent